src/main/java/frc/robot/Constants.java

```diff
@@ -135,6 +137,87 @@ public static final class BackRight { // Module 4
     }
   }
 
+  public static final class Arm {
+    public static final class Ports {
+      public static final int ARM_MOTOR_PORT = 16;
+      public static final int TELESCOPING_MOTOR_PORT = 17;
+      public static final int ENCODER_PORT = 28;
+    }
+
+    public static final double GRAVITY_CONTROL_PERCENT = 0.07;
+
+    public static final double ANGULAR_OFFSET = -8.75;
+
+    public static final class Setpoints {
+      public static final class ScoreLow {
```

Comment:
I think this will need refactoring into an object, so that it can be bundled into an object that encompasses a pose and arm state for scoring, but this refactor can come later.